tsconfig.json

```diff
@@ -0,0 +1,12 @@
+{
+  "compilerOptions": {
+    "strict": true,
+    "outDir": "./dist",
+    "target": "es2018",
+    "allowJs": true,
+    "checkJs": false,
+    "module": "commonjs"
+  },
+  "include": ["./lib/**/*"],
+  "exclude": ["node_modules"]
```

Comment:
Is this necessary if we are only including `./lib`? Or just a good precaution to have?

Reply:
It shouldn't be necessary in the long run, but without it the default setting is `"include": ["*.ts", "*.js"],` which currently grabs everything in `test`, `coverage`, and `bin`.

I think it would be nice to write tests, etc. in TypeScript, but this is just a baby step.